NavigationAdvancedSample/app/src/main/res/navigation/form.xml

```diff
@@ -22,18 +22,18 @@
     android:id="@+id/form"
     app:startDestination="@id/register">
 
-    <fragment
```

Comment:
I think we should try to keep some fragment destinations in our navgraphs to be able to support+test navigation between different destination types (we need that eventually in order to support compose destinations. Keeping fragments in here keeps us in line and prevents us from making "conductor specific" changes that breaks support of other destinations..)

Reply:
Yeah good point, I'll make a note of that, along with actually writing something which will massage the deeplinking flow. Ideally we should even test composable destinations alongside them too!

Reply:
Adding the FragmentNavigator to our navigatorProvider, and navigating from a controller navigation node to a fragment one works, but popping the fragment doesn't. Would need to spend more time getting that to work, if we ever really need that interop. Composable navigation isn't made via xml atm, so combining the different types of nodes would require the whole navigation graph to be made programatically. And in any case, initial interop could be hosting composable in a fragment or controller still